hr_contract_renew/views/hr_contract_view.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="hr_contract_view_form_inherit" model="ir.ui.view">
+        <field name="name">hr.contract.renew.inherit</field>
+        <field name="model">hr.contract</field>
+        <field name="inherit_id" ref="hr_contract.hr_contract_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//header">
+                <button
+                    name="%(hr_contract_renew.action_open_renew_contract_wizard)d"
+                    type="action"
+                    string="Renew Contract"
+                    class="oe_highlight"
+                    context="{'default_contract_id': id}"
+                    invisible="not id or date_end"
```

Comment:
```suggestion
                    states="draft,open"
                    invisible="not id"
```
Even if date_end is set (as in temporary contracts), renewal might still be needed.
Would it make sense to relax that condition? What do you think, @luisDIXMIT ?